### PR TITLE
[RHPAM-3709] - CVE-2021-26291 rhpam-7-businesscentral-rhel8-container…

### PR DESCRIPTION
**Pull request by spolti**
…: maven: Block repositories using http by default [rhpam-7]

Signed-off-by: spolti <fspolti@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Review by desmax74 — APPROVED** [review]

**Comment by spolti** [comment]
Hi @luck3y do you think we can merge this?

**Review by luck3y — APPROVED** [review]

**Comment by luck3y** [comment]
@spolti I think dropping the RHEL 7 stuff (scl) is probably a good idea, unless you've verified that Maven 3.8 has been added there as well (I can't find it.)

**Comment by spolti** [comment]
> @spolti I think dropping the RHEL 7 stuff (scl) is probably a good idea, unless you've verified that Maven 3.8 has been added there as well (I can't find it.)

seems to be here https://access.redhat.com/documentation/en-us/red_hat_software_collections/3/html/3.8_release_notes/chap-rhscl

 still on maven 3.6.
 
 Will remove it.,

**Comment by luck3y** [comment]
+1 from me, thanks @spolti!

File: jboss/container/maven/38/default/module.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+version: '3.8'
+description: Provides Maven v3.8 capabilities to an image.
+
+labels:
+- name: io.fabric8.s2i.version.maven
+  value: "3.8"
+
+envs:
+- name: JBOSS_CONTAINER_MAVEN_38_MODULE
+  value: /opt/jboss/container/maven/36/
```
Comment (spolti) [inline jboss/container/maven/38/default/module.yaml:12]:
```suggestion
  value: /opt/jboss/container/maven/38/
```

**Comment by spolti** [comment]
@luck3y I am going to merge and create a new tag, is there something else you would like to add to the 0.39.x branch before?

**Comment by luck3y** [comment]
All good from me, thanks!

**Pull request merged**